<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FinalizPopup</class>
 <widget class="QWidget" name="FinalizPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Customer Details</string>
  </property>
  <widget class="QFrame" name="container_frame">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>480</width>
     <height>430</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
     QFrame#container_frame {
       background-color: white;
       border: 2px solid #3855DB;
       border-radius: 20px;
     }
    </string>
   </property>
   <widget class="QFrame" name="title_bar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
      QFrame#title_bar {
        background-color: #3855DB;
        border-top-left-radius: 18px;
        border-top-right-radius: 18px;
      }
     </string>
    </property>
    <widget class="QLabel" name="label_title">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>440</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white; font-size: 18px; font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Finalize Payment - </string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_close">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>10</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
       QPushButton { background-color: transparent; color: white; font-size: 20px; border: none; }
       QPushButton:hover { background-color: rgba(255,255,255,0.2); border-radius: 15px; }
      </string>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="content_widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>461</width>
      <height>361</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_order_info">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>150</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Fredoka-SemiBold; font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Order Summary</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_iron">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>85</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Iron Only</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_dry">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Fast Dry:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_wash">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>35</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Wash:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_date">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>150</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Fredoka-SemiBold; font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Date Delivered:</string>
     </property>
    </widget>
    <widget class="QLabel" name="value_date">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>160</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_payment">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>150</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Fredoka-SemiBold; font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Payment Method:</string>
     </property>
    </widget>
    <widget class="QLabel" name="value_wash">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>35</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_process">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>60</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="value_procesd">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>60</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="value_subtotal_2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>360</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>â‚±0.00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_fold">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Fold:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_process_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>85</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_process_3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>110</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_grand_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>150</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Fredoka-SemiBold; font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Grand Total Amount:</string>
     </property>
    </widget>
    <widget class="QLabel" name="value_grand">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>190</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="cash">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Cash</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="card">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>260</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Card</string>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="b1">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>310</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
            background-color: #3855DB; /* Example background color */
            color: White; /* Example text color */
            border-radius: 20px; /* Adjust this value for desired roundness */
}
QPushButton:hover {
    background-color: #2746C3; /* Color when the mouse hovers over it */

}
QPushButton:pressed {
    background-color: #FFFFFF; /* Color when the button is clicked down */
	color: #3855DB;
	border-style: outset;
	border-width: 3px;
    border-radius: 20px; /* Adjust this value for desired roundness */
    border-color: #2746C3;
}</string>
     </property>
     <property name="text">
      <string>Complete Transaction</string>
     </property>
    </widget>
    <widget class="QPushButton" name="b2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
            background-color: #d9d9d9; /* Example background color */
            color: #737373; /* Example text color */
            border-radius: 20px; /* Adjust this value for desired roundness */
}
QPushButton:hover {
    background-color: #9c9b9b; /* Color when the mouse hovers over it */

}
QPushButton:pressed {
    background-color: #FFFFFF; /* Color when the button is clicked down */
	color: #9c9b9b;
	border-style: outset;
	border-width: 3px;
    border-radius: 20px; /* Adjust this value for desired roundness */
    border-color: #9c9b9b;
}</string>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QLabel" name="value_date_picked">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>135</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_datepicked">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>135</y>
       <width>150</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Fredoka-SemiBold; font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Date Picked:</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../images/Resc_rc.qrc"/>
 </resources>
 <connections/>
</ui>
